src/main/scala/org/allenai/scholar/metrics/metadata/Eval.scala

```diff
@@ -8,8 +8,9 @@ import org.allenai.scholar.{ Author, MetadataAndBibliography }
 
 import scala.collection.immutable
 import scala.io.Source
+import scala.util.{ Failure, Try }
 
-object Eval {
+object Eval extends FileParsing {
```

Comment:
Alternatively, define `FileParsing` as an object exposing `parseDir` function. Has-A vs Is-A.